tests/unit/modules/test_netconfig.py

```diff
@@ -0,0 +1,112 @@
+"""
+    :codeauthor: Gareth J. Greenaway <gareth@saltstack.com>
+"""
+
```

Comment:
This file should actually be in `tests/unit/states`

Reply:
oh man how did I do that wrong. Fixed